web-ui/src/main/resources/catalog/js/admin/DashboardController.js

```diff
@@ -147,6 +147,32 @@
         }
       });
 
+      $scope.infoSection = [
+        {
+          key: "catalogue",
+          label: "catalogInformation",
+          help: "installing-datadirectory"
+        },
+        {
+          key: "database",
+          label: "dbInformation",
+          help: "installing-db"
+        },
+        {
+          key: "index",
+          help: "installing-index"
+        },
+        {
+          key: "version"
+        },
+        {
+          key: "main",
+          label: "systemInformation"
+        },
+        {
+          key: "env"
```

Comment:
The final goal for this I guess is to have multiple instances 1 with the scheduler enabled and the others disabled.

I'm not sure how useful is to display this information in such scenario as depending on the instance accessed the value displayed will be different.

Also this kind of stuff raises some questions, for example what happens in the node with the scheduler enabled fails? Should be implemented in a future pull request some mechanism so other node picks up the scheduler, using JMS messages or similar?

Reply:
> The final goal for this I guess is to have multiple instances 1 with the scheduler enabled and the others disabled.

Yes.

> I'm not sure how useful is to display this information in such scenario as depending on the instance accessed the value displayed will be different.

Sure, but at least if you access the instance with scheduler on by its url you can check that the settings is properly enabled.


>Also this kind of stuff raises some questions, for example what happens in the node with the scheduler enabled fails? Should be implemented in a future pull request some mechanism so other node picks up the scheduler, using JMS messages or similar?

We are not here yet, but yes this is some further improvement we could plan.